.doc_gen/metadata/dynamodb_metadata.yaml

```diff
@@ -1492,6 +1504,17 @@ dynamodb_Scenario_PartiQLBatch:
   synopsis: query a &DDB; table by using batches of PartiQL statements.
   category: Scenarios
   languages:
+    .NET:
+      versions:
+        - sdk_version: 3
+          github: dotnetv3/dynamodb
+          excerpts:
+            - description:
+              snippet_tags:

```

Comment:
Per the EPIC, the batch should include GetItem. - https://github.com/awsdocs/aws-doc-sdk-examples/issues/2953

Reply:
That is exactly what I did. My README file includes only the name and link. I did not update the SoS but I did update the README file. Lines 22 to 24 include the three scenarios and their short descriptions.

Lines 107 to 170 of the PartiQLBatch_Methods class includes a call that selects multiple items from the table.

Not sure why you're not seeing the changes you requested but I just checked the code and the README file in the PR and the changes are there.